pkg/codegen/testing/test/testdata/output-funcs/nodejs/index.ts

```diff
@@ -5,27 +5,74 @@ import * as pulumi from "@pulumi/pulumi";
+export const funcWithListParam: typeof import("./funcWithListParam").funcWithListParam = null as any
+utilities.lazyLoadProperty(exports, "funcWithListParam", () => require("./funcWithListParam"));
+export const funcWithListParamOutput: typeof import("./funcWithListParam").funcWithListParamOutput = null as any
+utilities.lazyLoadProperty(exports, "funcWithListParamOutput", () => require("./funcWithListParam"));
+
+export { GetBastionShareableLinkArgs, GetBastionShareableLinkResult, GetBastionShareableLinkOutputArgs } from "./getBastionShareableLink";
+export const getBastionShareableLink: typeof import("./getBastionShareableLink").getBastionShareableLink = null as any
+utilities.lazyLoadProperty(exports, "getBastionShareableLink", () => require("./getBastionShareableLink"));
+export const getBastionShareableLinkOutput: typeof import("./getBastionShareableLink").getBastionShareableLinkOutput = null as any
+utilities.lazyLoadProperty(exports, "getBastionShareableLinkOutput", () => require("./getBastionShareableLink"));
+
+export { GetClientConfigResult } from "./getClientConfig";
+export const getClientConfig: typeof import("./getClientConfig").getClientConfig = null as any
+utilities.lazyLoadProperty(exports, "getClientConfig", () => require("./getClientConfig"));
+
+export { GetIntegrationRuntimeObjectMetadatumArgs, GetIntegrationRuntimeObjectMetadatumResult, GetIntegrationRuntimeObjectMetadatumOutputArgs } from "./getIntegrationRuntimeObjectMetadatum";
+export const getIntegrationRuntimeObjectMetadatum: typeof import("./getIntegrationRuntimeObjectMetadatum").getIntegrationRuntimeObjectMetadatum = null as any
+utilities.lazyLoadProperty(exports, "getIntegrationRuntimeObjectMetadatum", () => require("./getIntegrationRuntimeObjectMetadatum"));
+export const getIntegrationRuntimeObjectMetadatumOutput: typeof import("./getIntegrationRuntimeObjectMetadatum").getIntegrationRuntimeObjectMetadatumOutput = null as any
+utilities.lazyLoadProperty(exports, "getIntegrationRuntimeObjectMetadatumOutput", () => require("./getIntegrationRuntimeObjectMetadatum"));
+
+export { ListStorageAccountKeysArgs, ListStorageAccountKeysResult, ListStorageAccountKeysOutputArgs } from "./listStorageAccountKeys";
+export const listStorageAccountKeys: typeof import("./listStorageAccountKeys").listStorageAccountKeys = null as any
+utilities.lazyLoadProperty(exports, "listStorageAccountKeys", () => require("./listStorageAccountKeys"));
+export const listStorageAccountKeysOutput: typeof import("./listStorageAccountKeys").listStorageAccountKeysOutput = null as any
+utilities.lazyLoadProperty(exports, "listStorageAccountKeysOutput", () => require("./listStorageAccountKeys"));
+
+export { ProviderArgs } from "./provider";
+export type Provider = import("./provider").Provider;
+export const Provider: typeof import("./provider").Provider = null as any
+utilities.lazyLoadProperty(exports, "Provider", () => require("./provider"));
```

Comment:
This block illustrates the new code being harder to read than the old code.

Would it be an easy change to group all of the `utilities.lazyLoadProperty` calls together? Same with `export type` and `export const`, or just related syntactic forms in general.
Not a huge deal if not.

Reply:
Yeah that's a good idea! Still the new code will be hard to read, maybe we can emit a comment upfront saying why.